src/main/kotlin/LinkedList.kt

```diff
@@ -10,18 +10,54 @@ class LinkedList<T> {
     
     fun count(): Int = size
        
-    fun insert(index: Int, obj: T) {
-        TODO("Not yet implemented")
+    operator fun get(index: Int): T {
+        var current = head
+        var currentIndex = 0
+        while (current != null && currentIndex < index) {
+            current = current.nextNode
+            currentIndex++
+        }
+        return current
```

Comment:
Метод должен возвращать `T`, а ты возвращаешь `Node<T>?`. Т.е. тебе на `null` проверить надо и поле `value` в случае удачи вернуть.
```suggestion
        requireNotNull(current) { "Index out of boundaries." }
        return current.value
```